lib/ansible/modules/network/eos/eos_banner.py

```diff
@@ -91,12 +91,13 @@
 from ansible.module_utils.eos import load_config, run_commands
 from ansible.module_utils.eos import eos_argument_spec, check_args
 
+
 def map_obj_to_commands(updates, module):
     commands = list()
     want, have = updates
     state = module.params['state']
 
-    if state == 'absent' and 'text' in have.keys() and have['text']:
+    if state == 'absent' and 'text' in list(have.keys()) and have['text']:
```

Comment:
Not sure what you are trying to accomplish with list conversion here.
The purpose is to check if the key exist in the dictionary or not, that does not require list conversion. We can either do `key in have` or `key in have.keys()`.